ui/logviewer/Navigation.jsx

```diff
@@ -94,6 +95,15 @@ export default class Navigation extends React.PureComponent {
                 </a>
               </span>
             )}
+            <span>
+              <Button
+                className="nav-link btn-view-nav"
+                data-testid="log-lines-to-bug-filer"
+                onClick={copySelectedLogToBugFiler}
```

Comment:
Could you use (and modify if needed) the clipboard component that uses the clipboard API? It's in `ui/shared/Clipboard.jsx`

Reply:
Because this button doesn't copy to the clipboard, the benefit of sharing code would be only the UI (and it wouldn't match the component name `Clipboard`).